devices.js

```diff
@@ -10915,6 +10915,23 @@ const devices = [
         },
         exposes: [e.contact(), e.battery_low(), e.tamper(), e.temperature(), e.battery()],
     },
+    {
+        zigbeeModel: ['URC4460BC0-X-R'],
+        model: 'XHS2-UE',
+        vendor: 'Visonic',
```

Comment:
Googling shows me the sensor is made by Xfinity? (https://www.xfinity.com/support/articles/installing-battery-xhs2-ue-sensor)